### DISCUSSION TITLE
ImportError

### DISCUSSION THREAD

**Discussion by erwin-frohsinn** [tasmota/tasmotizer#63] [Q&A] [answered]
I followed option 2 to install tasmotizer and ran into an error.

My system: Kubuntu 20.04.1 LTS
Python 3.8.5
Running `tasmotizer.py` generated this error:
```
Traceback (most recent call last):
  File "/home/juergen/anaconda3/bin/tasmotizer.py", line 15, in <module>
    from PyQt5.QtNetwork import QNetworkRequest, QNetworkAccessManager, QNetworkReply
ImportError: /lib/x86_64-linux-gnu/libgssapi_krb5.so.2: undefined symbol: krb5_ser_context_init, version krb5_3_MIT

```

**Comment by feliperafael** [comment, marked as answer]
I had the same issue performing an installation directly. I created an env (python 3.6) in the conda and it worked.

`conda create -n tasmotaEnv python=3.6`

`conda activate tasmotaEnv`

`pip install tasmotizer`

apparently it's something related to this warning:

"After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts."

but I'm not sure
